src/LandIce/interface_with_mpas/Interface.cpp

```diff
@@ -87,12 +99,14 @@ void velocity_solver_solve_fo(int nLayers, int globalVerticesStride,
     const std::vector<double>& levelsNormalizedThickness,
     const std::vector<double>& elevationData,
     const std::vector<double>& thicknessData,
-    const std::vector<double>& betaData,
+          std::vector<double>& betaData,
     const std::vector<double>& bedTopographyData,
     const std::vector<double>& smbData,
     const std::vector<double>& stiffeningFactorData,
     const std::vector<double>& effectivePressureData,
+    const std::vector<double>& muData,
     const std::vector<double>& temperatureDataOnPrisms,
+    std::vector<double>& bodyForceOnBasalCell,
```

Comment:
I would call this bodyForceMagnitudeOnBasalCell, to clarify that it is the magnitude of the Body force